unittests/CppInterOp/ScopeReflectionTest.cpp

```diff
@@ -985,17 +985,16 @@ TEST(ScopeReflectionTest, GetBinaryOperator) {
 
   Cpp::Declare(code.c_str());
 
-  EXPECT_TRUE(Cpp::GetBinaryOperator(
-      Cpp::GetGlobalScope(), Cpp::BinaryOperator::Add, "MyClass", "MyClass"));
-  EXPECT_TRUE(Cpp::GetBinaryOperator(
-      Cpp::GetGlobalScope(), Cpp::BinaryOperator::Sub, "MyClass", "MyClass"));
-  EXPECT_TRUE(Cpp::GetBinaryOperator(
-      Cpp::GetGlobalScope(), Cpp::BinaryOperator::Add, "MyClass", "int"));
-  EXPECT_TRUE(Cpp::GetBinaryOperator(
-      Cpp::GetGlobalScope(), Cpp::BinaryOperator::Add, "int", "MyClass"));
-
-  EXPECT_FALSE(Cpp::GetBinaryOperator(
-      Cpp::GetGlobalScope(), Cpp::BinaryOperator::Add, "float", "MyClass"));
-  EXPECT_FALSE(Cpp::GetBinaryOperator(
-      Cpp::GetGlobalScope(), Cpp::BinaryOperator::Add, "MyClass", "float"));
+  std::vector<Cpp::TCppFunction_t> ops;
+
+  Cpp::GetBinaryOperator(Cpp::GetGlobalScope(), Cpp::BinaryOperator::Add, ops);
+  EXPECT_EQ(ops.size(), 3);
+  ops.clear();
+
+  Cpp::GetBinaryOperator(Cpp::GetGlobalScope(), Cpp::BinaryOperator::Sub, ops);
+  EXPECT_EQ(ops.size(), 1);
+  ops.clear();
+
+  Cpp::GetBinaryOperator(Cpp::GetGlobalScope(), Cpp::BinaryOperator::Mul, ops);
+  EXPECT_EQ(ops.size(), 0);
```

Comment:
Da we have an example involving argument dependent lookup?

Reply:
No. I have not implemented the argument dependent lookup with this PR.

I have made changes to take `std::vector&` as an argument and populate it with the operators. Removing the previous string comparisons to resolve the overloaded operator.
We discussed this after the monthly meeting the day before yesterday.